engine/crates/engine-v2/schema/src/builder/mod.rs

```diff
@@ -100,6 +100,7 @@ impl BuildContext {
                     directives: Default::default(),
                 },
             ],
+            field_to_parent_entity: vec![EntityId::Object(0.into()); 2],
```

Comment:
Why 2 entries of 0 here?  And is it worth a comment explaining

Reply:
ah not obvious in the PR, but it's just because I'm adding two fields `__schema` and `__type` before, so adding `Query` as their parent entity